src/dmclock_server.h

```diff
@@ -779,10 +804,11 @@ namespace crimson {
 			std::chrono::duration<Rep,Per> _idle_age,
 			std::chrono::duration<Rep,Per> _erase_age,
 			std::chrono::duration<Rep,Per> _check_time,
-			bool _allow_limit_break,
+			AtLimitParam at_limit_param,
 			double _anticipation_timeout) :
 	client_info_f(_client_info_f),
-	allow_limit_break(_allow_limit_break),
+	at_limit(get_or_default<AtLimit>(at_limit_param, AtLimit::Reject)),
```

Comment:
should we default to AtLimit::Reject?

Reply:
The standard dmclock algorithm uses AtLimit::Wait. So if there's a default, I think that should be it. We're trying to keep the library as generic as possible.

Also AtLimit::Reject puts additional requirements on the caller that neither Wait or Allow require (i.e., the caller has to do something different depending on whether the request is rejected or not). Additionally at this point we have so little experience with AtLimit::Reject.

Reply:
`Reject` is only chosen as a default when the type of the `at_limit_param` is not `AtLimit`, so it must be `RejectThreshold`

at a higher level, the Pull/Push queue constructors both provide a default argument for this:
`AtLimitParam at_limit_param = AtLimit::Wait`

that maintains the same behavior as the previous default:
`bool _allow_limit_break = false`